spec/api-browser-window-spec.js

```diff
@@ -1959,7 +1962,7 @@ describe('BrowserWindow module', function () {
         assert.equal(w.isResizable(), true)
       })
 
-      it('works for a frameless window', () => {
+      xit('works for a frameless window', () => {
```

Comment:
Right now it is mentioned in the PR's description. And I'm going to create an issue to fix the test if/after the PR is merged.